config/locales/de.yml

```diff
@@ -3221,6 +3221,12 @@ de:
             title:
               blank: 'Es muss ein Titel angegeben werden.'
               taken: 'Dieser Titel ist bereits vergeben.'
+        clicker_vote:
```

Comment:
Why are these translations introduced now? just curious :grinning:

Reply:
IIRC I built some invalid stuff in the creation of some factory. But the error message was not that it is invalid but that the translation of this error for ActiveRecord was missing. So I thought, better correct it right now...